cloudbuild/e2e-test.yaml

```diff
@@ -26,7 +26,7 @@ steps:
         mkdir -p ${COMMIT_SHA}/prediction/assets && \
         cp -r pipelines/pipelines/${_PIPELINE_TEMPLATE}/training/assets ${COMMIT_SHA}/training/ && \
         cp -r pipelines/pipelines/${_PIPELINE_TEMPLATE}/prediction/assets ${COMMIT_SHA}/prediction/ && \
-        gsutil cp -r ${COMMIT_SHA} ${_PIPELINE_PUBLISH_GCS_PATH}
+        gsutil cp -r ${COMMIT_SHA} ${_PIPELINE_PUBLISH_GCS_PATH}/${COMMIT_SHA}
```

Comment:
@felix-datatonic are you sure about this? Won't this create a nested directory?

Reply:
yes, tested successfully. in the prior case only the contents of `${COMMIT_SHA}` where copied to `${_PIPELINE_PUBLISH_GCS_PATH}`. feel free to test it different combinations and values in Cloud Build though.